styles.css

```diff
@@ -0,0 +1,716 @@
+  background-position-y: 3px;
+}
+
+.share {
+  background-image: url(/images/arrow-right.png);
+  background-size: 20px 20px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.save {
+  background-image: url(/images/bookmark.png);
+  background-color: transparent;
+  background-size: 30px 30px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.comments {
+  background-image: url(/images/comments.png);
+  background-color: transparent;
+  background-size: 30px 30px;
+  background-repeat: no-repeat;
+  background-position: center;
+  font-size: 10px;
+}
+
+/* ARTICLE IMG SECTION  */
+
+.article-img {
+  margin-top: 30px;
+  height: 400px;
+  width: 80%;
+}
+
+.article-img img {
+  max-width: 100%;
+  max-height: 100%;
+}
+
+.article-img2 {
+  display: flex;
+  width: 100%;
+}
+
+.article-img2 img {
+  max-width: 100%;
+  max-height: 100%;
+}
+
+.img-description {
+  width: 80%;
+  opacity: 0.8;
+  font-size: 19px;
+}
+
+.img-description2 {
+  width: 100%;
+  opacity: 0.8;
+  font-size: 17px;
+}
+
+.article-bottom {
+  margin-top: 20px;
+  font-size: 14px;
+}
+
+.article-bottom a {
+  text-decoration: underline;
+  color: inherit;
+}
+
+.author {
+  margin-top: 30px;
+  font-weight: bold;
+  width: 60%;
+}
+
+.author a {
+  text-decoration: none;
+  color: black;
+}
+
+.editor-pick-link {
+  margin-left: 10px;
+}
+
+.footer-nav a {
+  color: rgb(109, 105, 105);
+  margin-left: 10px;
+}
+
+.links a {
+  margin-bottom: 10px;
+  color: black;
+}
+
+.topic-img a {
+  text-decoration: none;
+  color: black;
+}
+
+.topic-section-title a {
+  color: black;
+}
+
+.editor-pick-link a {
+  color: black;
+  text-decoration: none;
+}
+
+.article-bottom a:hover {
+  text-decoration: none;
+}
+
+.writer {
+  color: grey;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 15px;
+}
+
+/* MAIN CONTAINER  */
+
+.container {
+  display: grid;
+  grid-template-columns: 1fr 3fr 1fr;
+  grid-auto-rows: minmax(100px, auto);
+  justify-content: center;
+}
+
+.content {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.date {
+  margin-top: 10px;
+  font-weight: normal;
+}
+
+.mid-content {
+  border-top: 1px solid #ebebeb;
+  font-family: "Times New Roman", Times, serif;
+  padding-top: 20px;
+  margin-top: 20px;
+  font-size: 24px;
+  width: 70%;
+  align-self: center;
+  line-height: 2rem;
+}
+
+p {
+  margin-bottom: 0;
+}
+
+article {
+  opacity: 0.8;
+  display: flex;
+  flex-direction: column;
+}
+
+/* SIDEBAR  */
+
+.sidebar {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 100%;
+}
+
+.editor {
+  font-weight: bold;
+  font-family: Arial, Helvetica, sans-serif;
+  border-bottom: 2px solid #ebebeb;
+  padding-bottom: 20px;
+  width: 300px;
+}
+
+.editor-pick {
+  margin-top: 20px;
+  display: flex;
+  align-items: center;
+  width: 100%;
+}
+
+.editor-pick-link a:hover {
+  text-decoration: underline;
+}
```

Comment:
## Aim to remain DRY
- [ ] you seem to be repeating your self on lines `#393`, `#529`, `#631` & `#677`. Do you mind merging the rules to avoid this?